### ISSUE TITLE
Does the option "-savefile" has bug?

### ISSUE THREAD

**Issue by SeekPoint** [reedscot/cvpr2016#4]
rzai@rzai00:~/prj/cvpr2016$ CUDA_VISIBLES_DEVICES=0 th train_sje_hybrid.lua -data_dir /media/rzai/ai_data/_reedscot/de_cvpr2016_cub.tar.gz -image_dir /media/rzai/ai_data/_reedscot/de_cvpr2016_cub.tar.gz/images -ids_file /media/rzai/ai_data/_reedscot/de_cvpr2016_cub.tar.gz/trainvalids.txt -learning_rate 0.0007 -symmetric 1 -max_epochs 200 -savefile sje_cub_c10_hybrid -num_caption 10 -gpuid 1 -print_every 10 2>&1 | tee yknote---train_sje_hybrid.lua---log
{
  image_dir : "/media/rzai/ai_data/_reedscot/de_cvpr2016_cub.tar.gz/images"
  seed : 123
  batch_size : 40
  num_caption : 10
  gpuid : 1
  symmetric : 1
  emb_dim : 1024
  image_noop : 1
  checkpoint_dir : "cv"
  bidirectional : 0
  randomize_pair : 0
  max_epochs : 200
  savefile : "sje_cub_c10_hybrid"
  print_every : 10
  data_dir : "/media/rzai/ai_data/_reedscot/de_cvpr2016_cub.tar.gz"
  image_dim : 1024
  init_from : ""
  doc_length : 201
  learning_rate_decay_after : 1
  grad_clip : 5
  avg : 0
  eval_val_every : 1000
  ids_file : "/media/rzai/ai_data/_reedscot/de_cvpr2016_cub.tar.gz/trainvalids.txt"
  nclass : 200
  cnn_dim : 256
  dropout : 0
  learning_rate : 0.0007
  learning_rate_decay : 0.98
  flip : 0
}
using CUDA on GPU 1...	
10/30000 (ep 0.067), loss=2.01, acc1=0.00, acc2=0.4246, g/p=2.7138e-02, t/b=5.54s	

*********many lines*********

990/30000 (ep 6.600), loss=0.52, acc1=30.00, acc2=24.6939, g/p=2.9552e-02, t/b=3.61s	
saving checkpoint to cv/lm_sje_cub_c10_hybrid_0.00070_1_10_/media/rzai/ai_data/_reedscot/de_cvpr2016_cub.tar.gz/trainvalids.txt.t7	
/home/rzai/torch/install/bin/luajit: cannot open <cv/lm_sje_cub_c10_hybrid_0.00070_1_10_/media/rzai/ai_data/_reedscot/de_cvpr2016_cub.tar.gz/trainvalids.txt.t7> in mode  w at /home/rzai/torch/pkg/torch/lib/TH/THDiskFile.c:670
stack traceback:
	[C]: at 0x7f0e06a66ad0
	[C]: in function 'DiskFile'
	/home/rzai/torch/install/share/lua/5.1/torch/File.lua:385: in function 'save'
	train_sje_hybrid.lua:268: in main chunk
	[C]: in function 'dofile'
	...rzai/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:145: in main chunk
	[C]: at 0x00406670
rzai@rzai00:~/prj/cvpr2016$ 
rzai@rzai00:~/prj/cvpr2016$ 
rzai@rzai00:~/prj/cvpr2016$

**Comment by SeekPoint** [comment]
--- a/train_sje_hybrid.lua
+++ b/train_sje_hybrid.lua
@@ -254,7 +254,7 @@ for i = 1, iterations do
         local val_loss = 0
         val_losses[i] = val_loss
 
-       local savefile  = string.format('%s/lm_%s_%.5f_%.0f_%.0f_%s.t7', opt.checkpoint_dir, opt.savefile, opt.learning_rate, opt.symmetric, opt.num_caption, opt.ids_file)
+       local savefile  = string.format('%s/lm_%s_%.5f_%.0f_%.0f.t7', opt.checkpoint_dir, opt.savefile, opt.learning_rate, opt.symmetric, opt.num_caption)
         print('saving checkpoint to ' .. savefile)
         local checkpoint = {}
         checkpoint.protos = protos
rzai@rzai00:/media/rzai/ai_data/prj/cvpr2016$ 



fixed!!!
